exercises/ex2.tex

```diff
@@ -103,7 +103,13 @@ \subsection*{Inference for binomial proportion (Computer)}
 > head(algae)
 \end{Sinput}
 \begin{Soutput}
-[1] 0 1 1 0 0 0
```

Comment:
This is not correct? It should be only a vector? Do you have the latest aaltobda package?

Reply:
If it's a dataframe I get the data frame format, so that's the output of the R code in my system. I freshly installed the aaltobda package yesterday, can you check if you get the vector?

Reply:
Actually, I think there is a mistake somewhere because the column name is "X0", and it should probably be "X" and the first element a 0 ???

Reply:
I get: 
```
> head(algae)
[1] 0 1 1 0 0 0
```

Reply:
I will try to reinstall the package from github again. The problem must be when reading the txt file. There may also be a mismatch of R versions that just print the data frame differently.

Reply:
Yes. Maybe it was a difference before and after Aki merged my PR.